### PR TITLE
Set user email in sentry when logged in

### PR DESCRIPTION
**Pull request by conbrad**
# Test Links:
[Landing Page](https://wps-pr-3658-e1e498-dev.apps.silver.devops.gov.bc.ca/)
[MoreCast](https://wps-pr-3658-e1e498-dev.apps.silver.devops.gov.bc.ca/morecast)
[Percentile Calculator](https://wps-pr-3658-e1e498-dev.apps.silver.devops.gov.bc.ca/percentile-calculator)
[MoreCast](https://wps-pr-3658-e1e498-dev.apps.silver.devops.gov.bc.ca/morecast)
[C-Haines](https://wps-pr-3658-e1e498-dev.apps.silver.devops.gov.bc.ca/c-haines)
[FireBat](https://wps-pr-3658-e1e498-dev.apps.silver.devops.gov.bc.ca/fire-behaviour-calculator)
[FireBat bookmark](https://wps-pr-3658-e1e498-dev.apps.silver.devops.gov.bc.ca/fire-behaviour-calculator?s=266&f=c5&c=NaN&w=20,s=286&f=c7&c=NaN&w=16,s=1055&f=c7&c=NaN&w=NaN,s=305&f=c7&c=NaN&w=NaN,s=344&f=c5&c=NaN&w=NaN,s=346&f=c7&c=NaN&w=NaN,s=328&f=c7&c=NaN&w=NaN,s=1399&f=c7&c=NaN&w=NaN,s=334&f=c7&c=NaN&w=NaN,s=1082&f=c3&c=NaN&w=NaN,s=388&f=c7&c=NaN&w=NaN,s=309&f=c7&c=NaN&w=16,s=306&f=c7&c=NaN&w=NaN,s=1029&f=c7&c=NaN&w=NaN,s=298&f=c7&c=NaN&w=NaN,s=836&f=c7&c=NaN&w=NaN,s=9999&f=c7&c=NaN&w=NaN)
[Auto Spatial Advisory (ASA)](https://wps-pr-3658-e1e498-dev.apps.silver.devops.gov.bc.ca/auto-spatial-advisory)
[HFI Calculator](https://wps-pr-3658-e1e498-dev.apps.silver.devops.gov.bc.ca/hfi-calculator)

### CODE REVIEW

File: web/src/features/auth/slices/authenticationSlice.test.ts [hunk 1]
```diff
@@ -137,6 +144,7 @@ describe('authenticationSlice', () => {
           token: testToken,
           idToken: testToken,
           idir: 'test@idir',
+          email: 'test@example.com',
```
Comment (dgboss) [inline web/src/features/auth/slices/authenticationSlice.test.ts:147]: nit: Can this use the `email` variable you defined on line 27?

File: web/src/features/auth/slices/authenticationSlice.test.ts [hunk 2]
```diff
@@ -159,6 +167,7 @@ describe('authenticationSlice', () => {
...
```
Comment (dgboss) [inline web/src/features/auth/slices/authenticationSlice.test.ts:170]: nit: Can this use the `email` variable you defined on line 27?

**Review by dgboss — APPROVED** [review]
Very nice!

File: web/src/features/auth/slices/authenticationSlice.test.ts (hunk 1 above)
Comment (conbrad) [reply web/src/features/auth/slices/authenticationSlice.test.ts]: Thanks fixed in https://github.com/bcgov/wps/pull/3658/commits/3a3c42d89187ad7d5ac0f2d8aa6aa3c3c5902465

File: web/src/features/auth/slices/authenticationSlice.test.ts (hunk 2 above)
Comment (conbrad) [reply web/src/features/auth/slices/authenticationSlice.test.ts]: Thanks fixed in https://github.com/bcgov/wps/pull/3658/commits/3a3c42d89187ad7d5ac0f2d8aa6aa3c3c5902465

**Review by brettedw — APPROVED** [review]
Nice!

**Pull request merged**